src/day-1-toy/args.py

```diff
@@ -4,14 +4,20 @@
 # Write a function f1 that takes two integer positional arguments and returns
 # the sum. This is what you'd consider to be a regular, normal function.
 
-#def f1(...
+def f1(num1, num2):
+    return num1 + num2
 
 print(f1(1, 2))
 
 # Write a function f2 that takes any number of iteger arguments and prints the
 # sum. Google for "python arbitrary arguments" and look for "*args"
 
```

Comment:
this works. There's also the built in sum function
def f2(*args):
    return(sum(args))